lib/cache.js

```diff
@@ -10,29 +10,8 @@ var Cache = module.exports = function (trie) {
 }
 
 Cache.prototype.put = function (key, val, fromTrie) {
-  var exists
-  if (val.exists) {
```

Comment:
Note that this `if` block always evaluates to `true`, so I removed it. If it's supposed to be something else, we should add that in later.